vsphere/resource_vsphere_resource_pool.go

```diff
@@ -0,0 +1,255 @@
+package vsphere
```

Comment:
It appears that tagging is missing from the schema and CRUD.

It looks like resource pools can be tagged, but not have custom attributes assigned to them, so tagging should be added (the appropriate schema fields), but not custom fields. Tests and docs would need to be updated as well.